app/models/item.rb

```diff
@@ -5,22 +5,47 @@ class Item < ActiveRecord::Base
   validates :text, :list_id, presence: true
 
   scope :incompleted, -> { where.not('done') }
+  scope :completed, -> { where('done') }
+  scope :for_today, -> {
+    where('completed_at >= ? AND completed_at <= ?',
```

Comment:
@vlasiak, I would name it something like `completed_today` as `for_today` doesn't show the intention.